responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -185,48 +185,31 @@ def compute(self):
         for cf_config in self._counterfactual_config_list:
             if not cf_config.is_computed:
                 cf_config.is_computed = True
-                try:
-                    dice_explainer = self._create_diceml_explainer(
-                        method=cf_config.method,
-                        continuous_features=cf_config.continuous_features)
-
-                    X_test = self._test.drop([self._target_column], axis=1)
-
-                    counterfactual_obj = \
-                        dice_explainer.generate_counterfactuals(
-                            X_test, total_CFs=cf_config.total_CFs,
-                            desired_class=cf_config.desired_class,
-                            desired_range=cf_config.desired_range)
-
-                    cf_config.counterfactual_obj = \
-                        counterfactual_obj
-                except Exception as e:
-                    cf_config.has_computation_failed = True
-                    cf_config.failure_reason = str(e)
-
-    def get(self, failed_to_compute=False):
-        """Return the computed counterfactual examples objects or failure reason.
-
-        :param failed_to_compute: Get the failure reasons if counterfactual
-                                  examples failed to compute.
-        :type failed_to_compute: bool
```

Comment:
Why is the error handling removed? This handles the error scenario from dice-ml library and records the reason within the counterfactual manager.

Reply:
none of other manger is doing this.
This way is pretty confused to end user
We should line up with other manger throw error during compute.

Reply:
Sorry, what is confusing to the other users? We can add a warning to signal to the user that the computation failed. 

If the other managers don't handle the error handling , they should address that issue.